pom.xml

```diff
@@ -147,6 +147,10 @@
     <version.com.ahome-it.lienzo-core>2.0.287-RELEASE</version.com.ahome-it.lienzo-core>
     <version.com.ahome-it.lienzo-tests>2.0.287-RELEASE</version.com.ahome-it.lienzo-tests>
 
+    <!-- JGIT 4.8.0.201706111038-r required by UberFire/AppFormer. Can be removed once IP BOM is upgraded.-->
+    <version.org.eclipse.jgit>4.8.0.201706111038-r</version.org.eclipse.jgit>
+    <version.org.apache.sshd>1.6.0</version.org.apache.sshd>
```

Comment:
Do we want/need this change only for 7.x? If so, we should probably branch ip-bom as well.